vsintegration/src/FSharp.Editor/Common/OpenDeclarationsGetter.fs

```diff
@@ -0,0 +1,476 @@
+﻿namespace Microsoft.VisualStudio.FSharp.Editor
```

Comment:
I don't really know what this file is doing.  My suspicion is that it should be part of the FCS API rather than in FSHarp.Editor. Please add a signature file to make the overall implemented operation clearer and allow future migration of the functionality behind the FCS API barrier.  Also I'm concerned that there is duplication here somehow with other operations in either FCS or FSharp.Editor